Tasks/ServiceFabricDeploy/ServiceFabricSDK/Utilities.ps1

```diff
@@ -204,4 +205,27 @@ function Merge-HashTables
     }
     $HashTableNew = $HashTableOld + $HashTableNew
     return $HashTableNew
+}
+
+function Get-TempDirectoryPath
```

Comment:
Consider:
The Service Fabric Update Manifests task also uses a temp directory (although it uses $ENV:AGENT_TEMPDIRECTORY) Probably would be prudent to put this in the Service Fabric common scripts and use it in both places.

Reply:
#fixed